src/compressed_tensors/quantization/quant_config.py

```diff
@@ -279,5 +278,38 @@ def requires_calibration_data(self):
 
         return False
 
+    def model_dump(self, *args, **kwargs):
+        # Call the parent dump first
+        data = super().model_dump(*args, **kwargs)
+
+        def _convert_dtypes_in_dict(d):
```

Comment:
> We serialize this information in the config except in these cases:
We set the zp_dtype to None if running symmetric quantization.
We set the scale_dtype to None if running dynamic or local quantization.

Why not serialize this information?

Reply:
To only retain compression / checkpoint information. 

I think as a user if I saw the zp_dtype but was running a symmetric model in vllm, I’d be confused

Reply:
Can we write this as a model validator so that this rule is enforced earlier rather than later? It's better to have a consistent and frozen data model when possible and as early as possible.